site/content/entry/citra-progress-report-2023-q1/index.md

```diff
@@ -0,0 +1,413 @@
+
+{{< figure src="cheats.png"
+    title="Getting more money could not be any easier!" >}}
+
+## Add support for custom textures and texture dumping ([#6144](https://github.com/citra-emu/citra/pull/6144)) by [LeviathaninWaves](https://github.com/LeviathaninWaves)
+
+Custom Textures are a way for modders to revamp the textures of a game to make them even more amazing than ever before! Due to the low resolution of the 3DS, games often used low-quality, compressed images for the models, so when Citra renders at upscaled resolutions, it’s still just not quite perfect.
+Texture Dumping is a way to gather all textures from a game whilst playing through it. No need to do anything fancy, all you need to do is play! These can be used for archival purposes or to create your own Custom Texture Packs to add right back into Citra.
+
+[Added to Citra Desktop](https://github.com/citra-emu/citra/pull/4868) all the way back in November 2019, first time developer [LevithaninWaves](https://github.com/LeviathaninWaves) tasked themselves with implementing this feature over on our Android build.
+
+To add Custom Textures to Citra, navigate to `Internal Storage -> citra-emu`, create a folder called `load`, another folder inside that called `textures` then a folder named with the TitleID of your game. These TitleIDs can be found on the [3DS releases database](http://3dsdb.com/) if you do not know it. 
+Place your custom textures inside this folder with the TitleID of your game. Dumped textures are stored in  `Internal Storage -> citra-emu -> dump -> textures`.
+Both Dump Textures and Custom Textures can be enabled in `Settings -> Graphics -> Utility`
+
+{{< figure src="textures.png"
+    title="Games can now be viewed so much clearer thanks to this!" >}}
+
+### Add internet permission to Android manifest ([#6167](https://github.com/citra-emu/citra/pull/6167)) by [PabloMK7](https://github.com/PabloMK7)
+
+This addition is basically what it says on the tin. Some homebrew apps require internet access to download data needed for them to work correctly, such as the Mario Kart 7 modpack [CTGP-7](https://ctgp-7.github.io/). As this is what is known as a “normal permission” granted from Android, any users of Citra do not need to agree to any additional privacy policies that may have been introduced due to this.
+
+NOTE: This PR *does not* mean that Citra Android has multiplayer access. It is only related to homebrew apps at the moment.
+
+### Allow opening in-emulation menu by pressing Back ([#6248](https://github.com/citra-emu/citra/pull/6248)) by [JosJuice](https://github.com/JosJuice)
+
+With the Android 13 update released on some devices came a big issue with Citra. No longer could you swipe down from the top to open the in-game menu! Instead, just the phone notification bar would be brought down.
+
+Due to this, you could no longer access Save States, Amiibo, Screen Layout, Overlay, FPS counter and so many more vital options to use Citra! This was a massive problem!
```

Comment:
I would remove "This was a massive problem!" ... just seems like unnecessary text since you already said there were issues and listed them.